src/Sarif.FunctionalTests/ConverterTestData/Semmle/results.csv.sarif

```diff
@@ -0,0 +1,1353 @@
+                  "endLine": 188,
+                  "endColumn": 39
+                }
+              }
+            }
+          ]
+        },
+        {
+          "level": "warning",
+          "message": "Variable [[\"reader\"|\"relative://src/main/java/org/apache/commons/io/input/Tailer.java:406:15061:31\"]] may be null here.",
+          "locations": [
+            {
+              "resultFile": {
+                "uri": "src/main/java/org/apache/commons/io/input/Tailer.java",
+                "region": {
+                  "startLine": 463,
+                  "startColumn": 25,
+                  "endLine": 463,
+                  "endColumn": 30
+                }
+              }
+            }
+          ]
+        },
+        {
+          "level": "warning",
+          "message": "Exceptions should not be dropped.",
+          "locations": [
+            {
+              "resultFile": {
+                "uri": "src/main/java/org/apache/commons/io/FileUtils.java",
+                "region": {
+                  "startLine": 1580,
+                  "startColumn": 11,
+                  "endLine": 1580,
+                  "endColumn": 41
+                }
+              }
+            }
+          ]
+        },
+        {
+          "level": "warning",
+          "message": "Exceptions should not be dropped.",
+          "locations": [
+            {
+              "resultFile": {
+                "uri": "src/main/java/org/apache/commons/io/monitor/FileAlterationMonitor.java",
+                "region": {
+                  "startLine": 189,
+                  "startColumn": 15,
+                  "endLine": 189,
+                  "endColumn": 56
+                }
+              }
+            }
+          ]
+        },
+        {
+          "level": "error",
+          "message": "This BOMInputStream is not always closed on method exit.",
+          "locations": [
+            {
+              "resultFile": {
+                "uri": "src/main/java/org/apache/commons/io/input/XmlStreamReader.java",
+                "region": {
+                  "startLine": 269,
+                  "startColumn": 36,
+                  "endLine": 269,
+                  "endColumn": 108
+                }
+              }
+            }
+          ]
+        },
+        {
+          "level": "error",
+          "message": "This BOMInputStream is not always closed on method exit.",
+          "locations": [
+            {
+              "resultFile": {
+                "uri": "src/main/java/org/apache/commons/io/input/XmlStreamReader.java",
+                "region": {
+                  "startLine": 270,
+                  "startColumn": 36,
+                  "endLine": 270,
+                  "endColumn": 81
+                }
+              }
+            }
+          ]
+        },
+        {
+          "level": "note",
+          "message": "Method doCopyFile ignores exceptional return value of File.setLastModified.",
```

Comment:
>doCopyFile [](start = 29, length = 10)

Semmle missed opportunity to both enquote symbol name and to provide link to declaration location. :) Someone should report out to them.